apps/docs/app/features/routes/ressurser/design-tokens/ShadowTokens.tsx

```diff
@@ -18,6 +49,53 @@ export function ShadowTokens(props: BoxProps) {
           Elevation 1, Elevation 2 og Elevation 3.
         </Text>
       }
-    ></SharedTokenLayout>
+    >
+      <Stack spacing={9}>
+        <ShadowTokensTable />
+      </Stack>
+    </SharedTokenLayout>
   );
 }
+
+type ShadowTokenTableProps = BoxProps;
+
+const ShadowTokensTable = (props: ShadowTokenTableProps) => {
+  const tokenFormatter = useTokenFormatter();
+  return (
+    <Box {...props}>
+      <Table variant="simple" colorScheme="grey">
+        <Thead>
+          <Tr>
+            <Th>Eksempel</Th>
+            <Th>Verdi</Th>
+            <Th>Kode</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {shadowTokens.map((token) => (
+            <Fragment>
```

Comment:
Du trenger ikke å bruke en Fragment her - kanskje bare fjerne den for å holde koden så enkel som mulig? 🤷‍♂️

Reply:
Done